lib/screens/home/search/service/service_search_results.dart

```diff
@@ -0,0 +1,33 @@
+import 'package:flutter/material.dart';
+import 'package:pet_app/model/pet.dart';
+import 'package:pet_app/model/service.dart';
+import 'package:pet_app/services/auth/auth_service.dart';
+import 'package:pet_app/services/services.dart';
+import 'package:pet_app/services/services/services_service.dart';
+import 'package:pet_app/widgets/service_list_view.dart';
+
+class ServicesSearchResults extends StatelessWidget {
+  final Pet forPet;
+  final ServicesService _serviceService = services.get<ServicesService>();
+
+  ServicesSearchResults({@required this.forPet});
+
+  @override
+  Widget build(BuildContext context) {
+    return FutureBuilder(
+      future: _serviceService.getServicesForPetType(forPet.type),
+      // ignore: missing_return
```

Comment:
Can be removed.

Reply:
What can be removed exactly? This part works as expected.

Reply:
The comment